### PR TITLE
Add KernelInfo UI

### PR DESCRIPTION
**Pull request by indig**
* Adds Info section above Symbol Browser
  This section is hidden when not in use
* Add Spark UI link to Info section
* Add info method to Kernel, support for loading info

Addresses #66

### CODE REVIEW

**Comment by indig** [comment]
screen shot 2019-02-21 at 6 58 36 pm

**Comment by jeremyrsmith** [comment]
Was planning to put this kind of thing under a "..." pop-out menu. But this is good for now!

File: polynote-frontend/polynote/messages.js [hunk 1]
```diff
@@ -600,10 +600,27 @@ export class KernelBusyState extends KernelStatusUpdate {
...
 KernelBusyState.codec = combined(bool, bool).to(KernelBusyState);
 
+export class KernelInfo extends KernelStatusUpdate {
+    static get msgTypeId() { return 3; }
+
+    static unapply(inst) {
+        return [inst.content];
+    }
+
+    constructor(content) {
```
Comment (jeremyrsmith) [inline polynote-frontend/polynote/messages.js]: Can we make this hold a dictionary of name/value pairs instead of a single string?

Comment (indig) [reply polynote-frontend/polynote/messages.js]: Sure, will do!

**Comment by indig** [comment]
> Was planning to put this kind of thing under a "..." pop-out menu. But this is good for now!

That sounds good too, I just wanted a quick n dirty solution for alpha but we can definitely iterate!

**Comment by indig** [comment]
Merge conflicts resolved and `...` collapsing added :)

**Pull request merged**